k8s/ithc/common/dg/dg-docassembly.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: dg-docassembly
+  namespace: dg
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.docassembly-api: glob:prod-*
+spec:
+  releaseName: dg-docassembly
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: dg-docassembly
+    version: 1.0.1
+  values:
+    docassembly-api:
+      replicas: 2
+      applicationInsightsInstrumentKey: "039faf8c-4682-432e-8fc5-f9735bb4b962"
```

Comment:
this should come from your vault ... and it doesn't look like the right key either

Reply:
I have requested this key as I am unable to access on Azure